src/WebJobs.Script.WebHost/Management/MeshInitServiceClient.cs

```diff
@@ -15,20 +18,23 @@ public class MeshInitServiceClient : IMeshInitServiceClient
         private readonly HttpClient _client;
         private readonly ILogger _logger;
         private readonly IEnvironment _environment;
+        private readonly IScriptWebHostEnvironment _webHostEnvironment;
 
         public MeshInitServiceClient(HttpClient client, IEnvironment environment,
+            IScriptWebHostEnvironment webHostEnvironment,
             ILogger<MeshInitServiceClient> logger)
         {
             _client = client ?? throw new ArgumentNullException(nameof(client));
             _logger = logger ?? throw new ArgumentNullException(nameof(logger));
             _environment = environment ?? throw new ArgumentNullException(nameof(environment));
+            _webHostEnvironment = webHostEnvironment ?? throw new ArgumentNullException(nameof(webHostEnvironment));
```

Comment:
>    _webHostEnvironment = webHostEnvironment ?? throw new ArgumentNullException(nameof(webHostEnvironment)); [](start = 8, length = 108)

are you just using the webhost environment to determine if we are in standbymode or not? Use StandbyOptions for that.